draft-ietf-ppm-dap.md

```diff
@@ -1276,6 +1280,12 @@ following generic checks.
       but during the collect sub-protocol. (See {{batch-validation}}.) If both
       checks succeed, the input share is not marked as invalid.
 
+1. Aggregators SHOULD NOT accept reports whose timestamp is too far into the
+   future. Implementors MAY provide for some small leeway, usually no more than
+   a few minutes, to account for clock skew. If a report is rejected for this
+   reason, the Aggregator SHOULD mark the input share as invalid with the error
+   `report_too_early`.
```

Comment:
```suggestion
1. Check if the report is too far into the future. Implementors can provide for
   some small leeway, usually no more than a few minutes, to account for clock
   skew. If a report is rejected for this reason, the Aggregator SHOULD mark
   the input share as invalid with the error `report_too_early`.
```
To both relax normative language around something that's hard to be normative about, and use consistent language with the other checks.